content/doc/book/scaling/architecting-for-scale.adoc

```diff
@@ -631,15 +620,13 @@ _secrets_ folder, your _identity.key_, your _secret.key_, and your
 _secret.key.not-so-secret_ file.
 
 [literal]
-+- identity.key   (RSA key pair that identifies an instance)
  +- secret.key     (used for various secure Jenkins operations)
  +- secret.key.not-so-secret  (used for validating _$JENKINS_HOME_ creation date)
  +- userContent    (files served in https://server/userContent/)
- +- secrets        (directory for the secret+key decryption)
-     +- hudson.util.Secret   (used for encrypting some Jenkins data)
```

Comment:
The hudson.util.Secret file is present on my installations that I use for bug verification. Refer to [JENKINS-75607](https://issues.jenkins.io/browse/JENKINS-75607) for an example of the script that starts from an empty directory and generates the hudson.util.Secret file in the secrets directory.

Please retain this file and its description